ui/cypress/integration/CreateVolume.feature

```diff
@@ -5,5 +5,5 @@ Feature: CreateVolume
      And I go to the bootstrap node by click on the bootstrap row in the list
      And I choose the Volumes tag
      And I go to create volume page by click on Create a New Volume button
-     And I fill out the create volume form with RawBlockDevice volume type
+     Then I fill out the create volume form with RawBlockDevice volume type and ckeck if the the volume I created is displayed on the volume list
```

Comment:
About the commit message: the title is confusing: `create a volume named a ` looks like an unfinished sentence ^^

It would be good to explain **why** we create a volume named `a`.
Maybe something like:

```
ui/tests: use `a` as Volume name

Because the list widget doesn't display the whole list of volumes, we
want to be sure that our volume is on the top of the list (so that we
can check its presence from Cypress).
Since the list is sorted alphabetically by name by default, by using
`a` as Volume name we're pretty sure that our volume will be on the
top.
```